data/search_synonyms.json

```diff
@@ -13,10 +13,89 @@
+    "Küche": ["Teilküche", "Anrichte", "Arbeitsraum für Verpflegung"],
+    "Anrichte": ["Teilküche", "Küche", "Arbeitsraum für Verpflegung"],
+    "Arbeitsraum für Verpflegung": ["Teilküche", "Küche", "Anrichte"],
+
+    "Kühlzelle": ["Kühlzellen", "Kühlraum", "Kühlkammer", "Kühlung", "Gefrierschrank", "Kühlschrank", "Kühllagerraum"],
+    "Kühlzellen": ["Kühlzelle", "Kühlraum", "Kühlkammer", "Kühlung", "Gefrierschrank", "Kühlschrank", "Kühllagerraum"],
+    "Kühlraum": ["Kühlzelle", "Kühlzellen", "Kühlkammer", "Kühlung", "Gefrierschrank", "Kühlschrank", "Kühllagerraum"],
+    "Kühlkammer": ["Kühlzelle", "Kühlzellen", "Kühlraum", "Kühlung", "Gefrierschrank", "Kühlschrank", "Kühllagerraum"],
+    "Kühlung": ["Kühlzelle", "Kühlzellen", "Kühlraum", "Kühlkammer", "Gefrierschrank", "Kühlschrank", "Kühllagerraum"],
+    "Gefrierschrank": ["Kühlzelle", "Kühlzellen", "Kühlraum", "Kühlkammer", "Kühlung", "Kühlschrank", "Kühllagerraum"],
+    "Kühlschrank": ["Kühlzelle", "Kühlzellen", "Kühlraum", "Kühlkammer", "Kühlung", "Gefrierschrank", "Kühllagerraum"],
+    "Kühllagerraum": ["Kühlzelle", "Kühlzellen", "Kühlraum", "Kühlkammer", "Kühlung", "Gefrierschrank", "Kühlschrank"],
+
+    "Müllager": ["Müll", "Müllraum", "Abfall", "Abfallsammelstelle", "Abfälle", "Entsorgungsraum", "Müllzwischenlager", "Müllsammelstelle"],
+    "Müll": ["Müllager", "Müllraum", "Abfall", "Abfallsammelstelle", "Abfälle", "Entsorgungsraum", "Müllzwischenlager", "Müllsammelstelle"],
+    "Müllraum": ["Müllager", "Müll", "Abfall", "Abfallsammelstelle", "Abfälle", "Entsorgungsraum", "Müllzwischenlager", "Müllsammelstelle"],
+    "Abfall": ["Müllager", "Müll", "Müllraum", "Abfallsammelstelle", "Abfälle", "Entsorgungsraum", "Müllzwischenlager", "Müllsammelstelle"],
+    "Abfallsammelstelle": ["Müllager", "Müll", "Müllraum", "Abfall", "Abfälle", "Entsorgungsraum", "Müllzwischenlager", "Müllsammelstelle"],
+    "Abfälle": ["Müllager", "Müll", "Müllraum", "Abfall", "Abfallsammelstelle", "Entsorgungsraum", "Müllzwischenlager", "Müllsammelstelle"],
+    "Entsorgungsraum": ["Müllager", "Müll", "Müllraum", "Abfall", "Abfallsammelstelle", "Abfälle", "Müllzwischenlager", "Müllsammelstelle"],
+    "Müllzwischenlager": ["Müllager", "Müll", "Müllraum", "Abfall", "Abfallsammelstelle", "Abfälle", "Entsorgungsraum", "Müllsammelstelle"],
+    "Müllsammelstelle": ["Müllager", "Müll", "Müllraum", "Abfall", "Abfallsammelstelle", "Abfälle", "Entsorgungsraum", "Müllzwischenlager"],
+    
+    "Gefahrenstoffraum": ["Gefahrenstofflager", "Gefahrengutraum", "Gefahrengutlager", "Giftlager"],
+    "Gefahrenstofflager": ["Gefahrenstoffraum", "Gefahrengutraum", "Gefahrengutlager", "Giftlager"],
+    "Gefahrengutraum": ["Gefahrenstoffraum", "Gefahrenstofflager", "Gefahrengutlager", "Giftlager"],
+    "Gefahrengutlager": ["Gefahrenstoffraum", "Gefahrenstofflager", "Gefahrengutraum", "Giftlager"],
+    "Giftlager": ["Gefahrenstoffraum", "Gefahrenstofflager", "Gefahrengutraum", "Gefahrengutlager"],
```

Comment:
Could you elaborate, why these synonyms should not be availible?
Searching where to get rid of the dangerous stuff sould be a good cause.
I dont see the danger you are seeing.

Reply:
I don't think this is a use case, because people working with dangerous goods should be knowing what they are doing. But if you think we should keep it, I'm also okay with that.

Reply:
While true, that people should get a good intro, sometimes they dont.
Probably minor anyway.
Originally I did not include this until I noticed what a mess the WZW is in naming these rooms.. Thought being that someone from Garching goes there for his Chemistry-Things..

Reply:
Just noticed, that TUM labels them Gefahr.. not Gefahren... Rooms
```suggestion
    "Gefahrstoffraum": ["Gefahrstofflager", "Gefahrgutraum", "Gefahrgutlager", "Giftlager"],
    "Gefahrstofflager": ["Gefahrstoffraum", "Gefahrgutraum", "Gefahrgutlager", "Giftlager"],
    "Gefahrgutraum": ["Gefahrstoffraum", "Gefahrstofflager", "Gefahrgutlager", "Giftlager"],
    "Gefahrgutlager": ["Gefahrstoffraum", "Gefahrstofflager", "Gefahrgutraum", "Giftlager"],
    "Giftlager": ["Gefahrstoffraum", "Gefahrstofflager", "Gefahrgutraum", "Gefahrgutlager"],
```

Reply:
Yes, we can leave them. I though about normalizing some namings in the data processing anyway, because too complex synonyms just slow down search performance and different naming schemes might irritate users (I'll create an issue)